vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItem.kt

```diff
@@ -122,6 +125,10 @@ abstract class RoomSummaryItem : VectorEpoxyModel<RoomSummaryItem.Holder>(R.layo
         holder.roomAvatarFailSendingImageView.isVisible = hasFailedSending
         renderSelection(holder, showSelected)
         holder.roomAvatarPresenceImageView.render(showPresence, userPresence)
+
+        if (useSingleLineForLastEvent) {
+            holder.subtitleView.setLines(1)
+        }
```

Comment:
Since items can be recycled, we need to add a `else` block here. But since the value is constant in the controller, I think this is fine.